drivers/base/kdgdb/kdcom.c

```diff
@@ -131,7 +131,12 @@ KdpPortInitialize(IN ULONG ComPortNumber,
     }
     else
     {
+        UCHAR Dummy;
+
         KdComPortInUse = KdComPort.Address;
+        /* Consume what could already be there */
+        while(CpGetByte(&KdComPort, &Dummy, FALSE, FALSE) == CP_GET_SUCCESS)
```

Comment:
`while<space>`

Reply:
Sure

Reply:
In fact, this is a leftover which can be deleted